### PR TITLE
Rami/checking grades

### LINKED ISSUE #1: Item1
Enable students to view the grades

So my students can keep updated with their grade and fix any issues that occur.

- [ ] make a log in for each student
- [x] make an area where they can look at the grade
- [x] give the teacher the ability to entet the grade
- [x] keep ungraded assignents hidden
- [ ] Have the program calculate the grade

acceptance 
1.  Only run on google chrome
2. Doesn't run from 3:00am -  4:00am

acceptance test
1. Students/ teacher successfully logs in to the website 
2. Correct dispay depending on user role
3. When student clicks show grades their grades show
4. Teacher can enter grades

### PR DESCRIPTION
**Pull request by rdari5**
Im pulling from the checking-grade  branch into the main branch

### CODE REVIEW

**Comment by rdari5** [comment]
I reviewed the pull request and the isabela file need more work.

File: Abhi [hunk 1]
```diff
@@ -0,0 +1 @@
+I made abhi
```
Comment (sjiang1) [inline Abhi:1]: hello review

File: Isabela [hunk 2]
```diff
@@ -0,0 +1,2 @@
+Hi
+Addding to the file
```
Comment (sjiang1) [inline Isabela:2]: Please add a period after line 2.

**Review by sjiang1 — CHANGES_REQUESTED** [review]
Please see my comments for Isabela.

**Review by sjiang1 — APPROVED** [review]
Approve!

**Comment by rdari5** [comment]
Approval for recent commit

**Pull request merged**